scripts/Common.ps1

```diff
@@ -1,4 +1,4 @@
-﻿##  Function: LogMessage
+﻿##  Function: Log-Message
```

Comment:
Good change but this function is used across the entire repository. Does this PR include an update to all references? If not, you may want to make that update in a follow up PR specific to that function.

Reply:
Yes Updated.